main.rb

```diff
@@ -0,0 +1,30 @@
+# !/usr/bin/env ruby
+
+require_relative 'app'
+
+class Main
+  def display_choices
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books. 📖'
+    puts '2 - List all people. 👤'
+    puts '3 - Create a person.'
+    puts '4 - Create a book.'
+    puts '5 - Create a rental.'
+    puts '6 - List all rentals for a given person ID.'
+    puts '7 - Exit. ❌'
+  end
+
+  def main
+    app = App.new
+    selection = nil
+    puts 'Welcome to the School Library App! 📖'
+    while selection != 'Exit'
+      display_choices
+      selection == app.select_choice
+    end
+    puts 'Thank you for using the School Library App! ✨'
+  end
+end
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224256281-49b317b0-5637-4abd-98fd-06d88a23ed61.png)

- [ ] Kindly add commands with the proper steps for using a ruby application because npm install has nothing to do with this ruby application and the steps you have written here are not useful for the viewer of your repo. It would be nice if you put information here regarding this ruby application to make your readme file descriptive and make your project attractive for the recruiters :blush: